torch/nn/attention/bias.py

```diff
@@ -283,11 +283,9 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
         """Defines the behavior of torch.nn.functional.scaled_dot_product_attention when the attn_bias is an AttnBias"""
         if kwargs is None:
             kwargs = {}
-        if func != torch.nn.functional.scaled_dot_product_attention:
-            raise NotImplementedError(
-                "CausalBias only supports scaled_dot_product_attention"
-            )
-        return cls._dispatch(*args, **kwargs)
+        if func == torch.nn.functional.scaled_dot_product_attention:
```

Comment:
Nit: should this be "is" instead of equality check?

Reply:
Good call, I'll update.